.github/workflows/bamboo-compatibility-check.yml

```diff
@@ -0,0 +1,123 @@
+name: 'Bamboo compatibility update check'
+
+on:
+  schedule:
+    # Monday 8am AEST (Sunday 10pm UTC)
+    - cron: '0 22 * * 0'
+
+  workflow_dispatch:
+
+env:
+  GH_TOKEN: ${{ secrets.INTEGRATIONS_FNM_BOT_TOKEN }}
+  GITHUB_TOKEN: ${{ secrets.INTEGRATIONS_FNM_BOT_TOKEN }}
+
+jobs:
+  check-for-update:
+    name: 'Check for new version'
+    runs-on: 'ubuntu-latest'
+
+    outputs:
+      update-available: ${{ steps.version-check.outputs.update-available }}
+      compatible-version: ${{ steps.version-check.outputs.current-compatible-version }}
+      bamboo-version: ${{ steps.version-check.outputs.latest-bamboo-version }}
+      gh-issue-url: ${{ steps.create-issue.outputs.issue-url }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: 'Check for new version of Bamboo'
+        id: version-check
+        run: |
+          # get the version our plugin is compatible with
+          currentCompatibleVersion=$(curl -sS -X GET https://marketplace.atlassian.com/rest/2/addons/com.octopus.bamboo/versions/latest | jq '.compatibilities[0].hosting.server.max.version')
+          echo $currentCompatibleVersion
+          # get latest available version of Bamboo
+          latestBambooVersion=$(curl -sS -X GET https://marketplace.atlassian.com/rest/2/products/key/bamboo/versions/latest | jq '.name')
+          echo $latestBambooVersion
+          echo "::set-output name=update-available::true"
+          echo "::set-output name=current-compatible-version::$currentCompatibleVersion"
+          echo "::set-output name=latest-bamboo-version::$latestBambooVersion"
+
+  create-issue:
```

Comment:
typically jobs imply running things in parallel, or sequentially but on different runners (e.g. stage 1 linux stage 2 windows)

This looks like a linear flow here with dependencies between each one; would it be simpler just to have a single job with multiple steps? The conditional/if stuff would work the same

Reply:
I had it that way initially but if anything fails, you have to drill into the job then find the step that failed and drill into that. Less clicks having them as separate jobs :grin:

Reply:
LOL fair enough

Reply:
It also makes for a nicer job summary 😄 

![image](https://user-images.githubusercontent.com/1035315/182963479-73d937f8-a81b-4476-9347-28f5a6df5f8e.png)